en/deploy-br-federation-manager.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Deploy BR Federation Manager on Kubernetes
+summary: Learn how to deploy BR Federation Manager on Kubernetes.
+---
+
+# Deploy BR Federation Manager on Kubernetes
+
+## Prerequisites
+
+Before deploy BR Federation Manager on Kubernetes, you should satisfy conditions below:
+
+* you have multiple Kubernetes clusters
+* Every Kubernetes clusters can communicate with each other by network
+* All the Kubernetes clusters that serve as data planes have deployed TiDB-Operator
+
+## Deploy BR Federation Manager
+
+### Generate Kubeconfig files in all Data Planes
+
+BR federation manager manages Kubernetes clusters of data planes by accessing their API servers.
+So it needs kubeconfing files to authenticate the API servers. The user or service account in the kubeconfig file need
+have all the permissions of **backups.pingcap.com, restores.pingcap.com** CRD at least.
+
+You can get kubeconfig files from the Kubernetes cluster administrator. But we also provide a way to generate
+kubeconfig files by yourself if you have the permission to access all the data planes.
+
+#### Create Resources about RBAC
+
+We should create the resources below in every data plane to allow br federation manager to manipulate backup and restore CR.
+
+{{< copyable "shell-regular" >}}
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: br-federation-member
+  namespace: tidb-admin
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
qq: why do we need clusterRole here? can we use Role?

Reply:
You can use `Role`, if you only create backup/restore in one namespace. The reason why I use `ClusterRole` in the doc is that we aren't sure whether the user creates backup/restore in one namespace or in multiple namespaces. To avoid the error about permission, I write `ClusterRole` here.